cosmosis/output/in_memory_output.py

```diff
@@ -33,7 +37,12 @@ def __getitem__(self, key_or_index):
     def from_options(cls, options, resume=False):
         if resume:
             raise ValueError("Cannot resume from in-memory output")
-        return cls()
+        apply_blinding_offsets = utils.boolean_string(options.get('apply_blinding_offsets', False))
```

Comment:
There's probably no need to have both a T/F option and a choice for the file - you can just see if the file name is set at all and if so use it, and if it's blank then don't apply any offset.